Ast/src/Parser.cpp

```diff
@@ -26,10 +26,14 @@ LUAU_DYNAMIC_FASTFLAGVARIABLE(LuaReportParseIntegerIssues, false)
 
 LUAU_FASTFLAGVARIABLE(LuauAlwaysCaptureHotComments, false)
 
+LUAU_FASTFLAGVARIABLE(LuauInterpolatedStringBaseSupport, true)
+
 bool lua_telemetry_parsed_out_of_range_bin_integer = false;
 bool lua_telemetry_parsed_out_of_range_hex_integer = false;
 bool lua_telemetry_parsed_double_prefix_hex_integer = false;
 
+#define ERROR_INVALID_INTERP_DOUBLE_BRACE "Double braces are not permitted within interpolated strings. Did you mean '\\{'?"
```

Comment:
This should be something like `const char* kErrorInvalidInterpDoubleBrace`

Reply:
When using this, I then get the error:

```
Ast/src/Parser.cpp:2701:80: warning: format not a string literal and no format arguments [-Wformat-security]
 2701 |             return reportExprError(location, {}, kErrorInvalidInterpDoubleBrace);
```

Reply:
Ah, whelp.